dist/releases/4.19/config.toml

```diff
@@ -0,0 +1,433 @@
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-azurerm",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-azurestack",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-google",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-ibm",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-ignition",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-libvirt",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-local",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-nutanix",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-openstack",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-ovirt",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-time",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-vsphere",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform-provider-vsphereprivate",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-aws",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-azurerm",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-azurestack",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-google",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-ibm",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-ignition",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-libvirt",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-local",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-nutanix",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-openstack",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-ovirt",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-time",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-vsphere",
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_arm64/terraform-provider-vsphereprivate",
+]
+
+[[payload.ose-installer-terraform-providers-container.ignore]]
+error = "ErrGoNoCgoInit"
+files = [
+  "/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform",
```

Comment:
We could benefit from expressing this whole list as `"/go/src/github.com/openshift/installer/terraform/bin/linux_amd64/terraform*"`

Reply:
@mrunalp we would want to do that for other files as well right? I can raise a follow up PR, since the changes do not relate to the branching task here, if that works

Reply:
While the existing implementation isn't very dry, it is explicit. Is this list of files everything under `/bin/linux_amd64/terraform*`? Or are there some files under here that aren't subject to ErrGoNoCgoInit?

Reply:
@rhmdnd I agree with you, if there are files that should not be subject to the exclude, then we should not use the wildcard. Not sure how we can verify that though